### PR TITLE
dosage: Init at 2016.03.17

### PR DESCRIPTION
**Pull request by Baughn**
###### Things done:
- [ ] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [ ] NixOS
  - [X] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

_Please note, that points are not mandatory, but rather desired._

### CODE REVIEW

**Comment by joachifm** [comment]
Looks like it's missing some build-time dependencies (import errors during testing).

**Comment by Baughn** [comment]
I hadn't tested it with every version of Python that the evaluator does, it seems. Fixed now.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -1966,6 +1966,28 @@ in modules // {
...
+      rev = "1af022895e5f86bc43da95754c4c4ed305790f5b";
+      sha256 = "1bkqhlzigy656pam0znp2ddp1y5sqzyhw3c4fyy58spcafldq4j6";
+    };
+    buildInputs = with self; [ pytest ];
+    propagatedBuildInputs = with self; [ requests2 lxml pbr ];
+    # prompt_toolkit doesn't work on 3.5 on OSX.
+    doCheck = !isPy35;
+
+    meta = {
+      description = "A comic strip downloader and archiver.";
```
Comment (joachifm) [inline pkgs/top-level/python-packages.nix]: Please delete the trailing dot from the description

**Comment by joachifm** [comment]
Apart from the obligatory nitpick, I think this is ready to go.

**Comment by Baughn** [comment]
Done.

**Pull request merged**